### ISSUE TITLE
The accuracy of efficient-edge model is inconsistent

### ISSUE THREAD

**Issue by maxadda** [tensorflow/tpu#489]
The image data set used is the Imagenet 2012 validation set, a total of 50,000 pictures. Using eval_ckpt_main.py script to test S-M-L three models, the accuracy is as follows:

S model: top1-77.61%, top5-93.77%
M model: top1-78.98%, top5-94.5%
L model: top1-80.47%, top5-95.19%

All the tests are completed according to the instructions of the official website. What are the causes of the errors? I can only guess the impact of bfloat16 and bfloat32?
Platform:
tf version: 1.13
GPU : 1080Ti

**Comment by mingxingtan** [comment]
Your results are slightly different from out results (from TPU):

efficientnet-edgetpu-S | 77.23% | 77.0 %
efficientnet-edgetpu-M | 78.69 | 78.6 %
efficientnet-edgetpu-L | 80.62 | 80.2%

I am not sure about the difference (bfloat might be one of the reasons), but the difference seems small. Are you using float32 on GPU?  If possible, could you also help eval them on CPU?  Thanks!
